common/scripts/install-run-rush.js

```diff
@@ -1,3 +1,7 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
could you revert the changes in here, and commit using `no-verify` to prevent addition of bentley copyright headers?

same thing for the install-run.js script

Reply:
fixed